### PR TITLE
Embed yt videos

### PR DESCRIPTION
**Pull request by mkwhalen**
- Documentation content changes

### CODE REVIEW

**Comment by agneszitte-nventive** [comment]
@mkwhalen While looking at your branch on Github, here is how it will be displayed 
image

Seems it's not possible to embed youtube video in GitHub markdown using iframe.
But you could use and image with the link like that:
```
[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg)](https://www.youtube.com/watch?v=YOUTUBE_VIDEO_ID_HERE)
```
(You also have this: http://embedyoutube.org/)

**Comment by mkwhalen** [comment]
@agneszitte-nventive thanks for catching that! All se!

File: doc/articles/get-started.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ If XAML Intellisense isn't working on a freshly-created project, try the followi
 3. Close and reopen Visual Studio.
 4. Reopen XAML documents.
 
+#### Video Tutorial
+[![](http://img.youtube.com/vi/hQcKsoxaAHo/0.jpg)](http://www.youtube.com/watch?v=hQcKsoxaAHo "")
```
Comment (agneszitte-nventive) [inline doc/articles/get-started.md:69]:
```suggestion
[![Getting Started with Uno Platform and Visual Studio for Windows](http://img.youtube.com/vi/hQcKsoxaAHo/0.jpg)](http://www.youtube.com/watch?v=hQcKsoxaAHo)
```

File: doc/articles/get-started.md [hunk 2]
```diff
@@ -137,6 +140,9 @@ Here's how to do this:
 
 You should now be in the same configuration as the generated template files.
 
+#### Video Tutorial
+[![](http://img.youtube.com/vi/O9qgw1Q445M/0.jpg)](http://www.youtube.com/watch?v=O9qgw1Q445M "")
```
Comment (agneszitte-nventive) [inline doc/articles/get-started.md:144]:
```suggestion
[![Creating WebAssembly Project with Uno Platform and Visual Studio Code](http://img.youtube.com/vi/O9qgw1Q445M/0.jpg)](http://www.youtube.com/watch?v=O9qgw1Q445M)
```

File: doc/articles/get-started.md [hunk 3]
```diff
@@ -172,6 +178,9 @@ Building for WebAssembly takes a few more steps than iOS and Android:
 3. In the terminal, navigate to your build output. This will typically be: `yourProject.Wasm > bin > Debug > netstandard2.0 > dist > server.py` Run the `server.py` program.
 4. In your browser, open localhost:8000. 
 
+#### Video Tutorial
+[![](http://img.youtube.com/vi/ESGJr6kHQg0/0.jpg)](http://www.youtube.com/watch?v=ESGJr6kHQg0 "")
```
Comment (agneszitte-nventive) [inline doc/articles/get-started.md:182]:
```suggestion
[![Getting Started with Uno Platform and Visual Studio for Mac](http://img.youtube.com/vi/ESGJr6kHQg0/0.jpg)](http://www.youtube.com/watch?v=ESGJr6kHQg0)
```

File: doc/articles/get-started.md [hunk 4]
```diff
@@ -223,6 +232,9 @@ Alternatively, you can use a tool like VNC to run the simulator on a mac.
 You will be able to build the UWP project, however, Rider currenly does not support debugging or deploying for UWP.   
 ![run-uwp-rider](Assets/quick-start/run-uwp-rider.JPG)  
 
+#### Video Tutorial
+[![](http://img.youtube.com/vi/HgwL0al5bfo/0.jpg)](http://www.youtube.com/watch?v=HgwL0al5bfo "")
```
Comment (agneszitte-nventive) [inline doc/articles/get-started.md:236]:
```suggestion
[![Getting Started with Uno Platform and JetBrains Rider](http://img.youtube.com/vi/HgwL0al5bfo/0.jpg)](http://www.youtube.com/watch?v=HgwL0al5bfo)
```

**Pull request merged**